scheduler/src/cook/tools.clj

```diff
@@ -950,16 +953,21 @@
                 [usage' (below-quota? quota usage')]))]
       (filter-sequential filter-with-quota usage queue))))
 
-(defn filter-pending-jobs-for-autoscaling
+(defn filter-pending-jobs-for-quota
   "Lazily filters jobs to those that should be considered
   for autoscaling purposes. Note that this is used in two
   places: the /queue endpoint (Mesos only) and as an
-  argument to scheduler/trigger-autoscaling! (k8s only)."
+  argument to scheduler/trigger-autoscaling! (k8s only).
```

Comment:
This is used in more than 2 places now, right?

Reply:
Comment fixed.